### DISCUSSION TITLE
Question about the source code of base_mechanics.py

### DISCUSSION THREAD

**Discussion by fsenoner** [pybamm-team/PyBaMM#3149] [Q&A]
Dear Pybamm team,

may I ask one question about the source code of the base_mechanics.py file in the particle_mechanics folder:

```
# stress evaluated at the surface of the particles
# Ai2019 eq [7] with r=R
stress_r_surf = pybamm.Scalar(0)
```
Why is the mechanical stress in radial direction set to 0?

Thank you

**Comment by brosaplanella** [comment]
I am not familiar with this model, but it seems to be a consequence of the equations (see equation [7] of the paper: https://iopscience.iop.org/article/10.1149/2.0122001JES/pdf). If I understand it is a direct consequence of the boundary condition in Table II.
